lib/imap.php

```diff
@@ -85,12 +85,19 @@ public function checkPassword($uid, $password) {
 					$groups[] = $pieces[1];
 		}
 
-		$protocol = ($this->sslmode === "ssl") ? "imaps" : "imap";
-		$url = "{$protocol}://{$this->mailbox}:{$this->port}";
 		$ch = curl_init();
-		if ($this->sslmode === 'tls') {
-			curl_setopt($ch, CURLOPT_USE_SSL, CURLUSESSL_ALL);
+
+		if ($this->sslmode !== 'tls') {
```

Comment:
shouldn't people with `tls` set use STARTTLS by default? - this would exclude them, right?